client/app/containers/test-run-info/test-run-info.controller.js

```diff
@@ -4,10 +4,10 @@ import ImagesViewerController from '../../components/modals/images-viewer/images
 
 const testRunInfoController = function testRunInfoController($scope, $rootScope, $mdDialog, $interval,
     $filter, $anchorScroll, $location, $timeout, $q,
-    elasticsearchService, TestRunService, UtilService,
-    ArtifactService, $stateParams, OFFSET,
-    API_URL, $state, TestRunsStorage,
-    TestService, $transitions, pageTitleService) {
+    elasticsearchService, testsRunsService, UtilService,
+    ArtifactService, $stateParams, OFFSET, API_URL,
+    $state, TestRunsStorage, TestService, $transitions,
+    pageTitleService) {
```

Comment:
Please split to the one column and add the trailing comma;
And please double-check dependencies: if nothing is missed and there are no unused ones.

Reply:
fixed